tests/test_agent_loop.py

```diff
@@ -145,7 +147,7 @@ def _handle(self, envelope: Envelope) -> None:
 
     def put_inbox(self, msg: Any) -> None:
         """Add a message to inbox."""
-        self._inbox.put_nowait(msg)
+        self._inbox.put_nowait(msg)  # type: ignore
```

Comment:
why type ignore?; check inbox not none

Reply:
just test tool, it allow any kind of data to be passed here.